rbc/request_internal_test.go

```diff
@@ -0,0 +1,45 @@
+package rbc
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestValReqRepository_Find(t *testing.T) {
+	valRepo, _ := NewValReqRepository()
+	valStruct := ValRequest{nil, nil, nil}
+	valRepo.Save("ID1", &valStruct)
+	if valRepo.Find("ID1") != &valStruct {
+		t.Fatalf("request %v is not found.", valRepo.recv["ID1"])
+	}
+}
+func TestValReqRepository_FindAll(t *testing.T) {
+	valRepo, _ := NewValReqRepository()
+	valStruct := ValRequest{nil, nil, nil}
+	valRepo.Save("ID1", &valStruct)
+	valRepo.Save("ID2", &valStruct)
+	if len(valRepo.FindAll()) != 2 {
+		t.Fatalf("request %v,%v is not found all.", valRepo.recv["ID1"], valRepo.recv["ID2"])
+	}
+	for _, request := range valRepo.FindAll() {
+		valReque, _ := request.(*ValRequest)
+		if !reflect.DeepEqual(valReque, &valStruct) {
+			t.Fatalf("request %v,%v is not found all.", valRepo.recv["ID1"], valRepo.recv["ID2"])
+		}
+	}
+}
+func TestValReqRepository_Save(t *testing.T) {
+	valRepo, _ := NewValReqRepository()
+	valstruct := ValRequest{nil, nil, nil}
+	valRepo.Save("ID1", &valstruct)
+	_, ok := valRepo.recv["ID1"]
+	if !ok {
+		t.Fatalf("request %v is not saved", valRepo.recv["ID1"])
+	}
+}
+func TestValReqRepository_NewValReqRepository(t *testing.T) {
+	_, err := NewValReqRepository()
+	if err != nil {
+		t.Fatalf("ValReqRepository is not created.")
```

Comment:
plz start with lower-case, for example

*val request repository is not created*